src/zzz_od/backend/mcp/config_app.py

```
@@ -0,0 +1,247 @@
+    _ro_item_fields_for,
+    all_entries,
+    get_entry,
+)
+
+
+def make_add_config_item(backend: ZzzBackendContext) -> Callable:
+    """构造 ``add_config_item`` tool。"""
+    async def add_config_item(
+        app_id: Annotated[str, Field(description="app 配置 id,如 charge_plan")],
+        list_field: Annotated[str, Field(description="列表字段名(如 plan_list/app_list)")] = 'plan_list',
+        item_dict: Annotated[dict, Field(description="列表项 dict(经 from_dict 反序列化)")] = None,
```

Ruff (0.15.21)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)